build.gradle

```diff
@@ -26,12 +26,6 @@ allprojects {
 artifactoryPublish.skip = true
 
 artifactory {
-    contextUrl = 'https://oss.jfrog.org'
-    resolve {
-        repository {
-            repoKey = 'libs-release'
-        }
-    }
```

Comment:
Removing this causes the build tio fail, if executed without jfrog-cli:
```
$ gradle clean install
Starting a Gradle Daemon (subsequent builds will be faster)

> Configure project :artifactory-java-client-services
Creating $fileName

> Task :artifactory-java-client-api:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':artifactory-java-client-api:compileJava'.
> Could not resolve all files for configuration ':artifactory-java-client-api:compileClasspath'.
   > Cannot resolve external dependency org.apache.httpcomponents:httpcore:4.4.1 because no repositories are defined.
     Required by:
         project :artifactory-java-client-api
   > Cannot resolve external dependency org.apache.httpcomponents:httpclient:4.5 because no repositories are defined.
     Required by:
         project :artifactory-java-client-api
   > Cannot resolve external dependency commons-lang:commons-lang:2.3 because no repositories are defined.
     Required by:
         project :artifactory-java-client-api
   > Cannot resolve external dependency com.fasterxml.jackson.core:jackson-core:2.9.5 because no repositories are defined.
     Required by:
         project :artifactory-java-client-api
   > Cannot resolve external dependency com.fasterxml.jackson.core:jackson-databind:2.9.5 because no repositories are defined.
     Required by:
         project :artifactory-java-client-api
   > Cannot resolve external dependency com.fasterxml.jackson.core:jackson-annotations:2.4.6 because no repositories are defined.
     Required by:
         project :artifactory-java-client-api
```